### PR TITLE
Tweak itch.io profile

### PR DESCRIPTION
**Pull request by SpotComms**
I've not used itch.io but should we also remove whitelist? Do games actually store their configs in ~/.config/itch, or is itch purely a launcher and games can store wherever like Steam?

### CODE REVIEW

File: etc/itch.profile [hunk 1]
```diff
@@ -22,12 +23,13 @@ nogroups
...
+novideo
 protocol unix,inet,inet6,netlink
 seccomp
 shell none
 
 private-dev
 private-tmp
 
-noexec ${HOME}
+#noexec ${HOME}
```
Comment (Fred-Barclay) [inline etc/itch.profile]: Looks good! I've not used itch.io either, but do we need to comment out `noexec ${HOME}`?

Comment (SpotComms) [reply etc/itch.profile]: Assuming its a launcher, its downloading games and needs to be able to run them.

**Comment by smitsohu** [comment]
I also have not used itch, but I looked a bit on their page and I found this: https://itch.io/docs/itch/using/sandbox/linux.html

If I understand that right, shouldn't there be a comment in the profile that itch.io already ships Firejail and a Firejail profile?

**Comment by SpotComms** [comment]
@smitsohu oh wow, thats nice... except how many people have /home mounted **without** `nosuid`?  But yes we should probably add a comment and also not let firecfg enable it by default.

**Comment by Ferroin** [comment]
@SpotComms Well, to start with, anybody who has `/home` as part of the root filesystem, which I would guess is a significant majority of desktop Linux users considering that that's default behavior in almost every major distribution's installer, and is trivial to do with those that don't have that behavior.

**Comment by netblue30** [comment]
merged, thanks.

**Comment by pizzadude** [comment]
I wrote the itch.io profile because the itch.io built-in firejail doesn't work for me.

**Pull request merged**